src/test/java/com/easypost/ReportTest.java

```diff
@@ -64,16 +66,59 @@ public static void setup() throws EasyPostException {
         globalTrackerReport = Report.create(trackerParams);
     }
 
+    /**
+     * Test creating a report with additional columns.
+     *
+     * @throws EasyPostException when the request fails.
+     */
+    @Test
+    public void testCreateReportWithAdditionalColumns() throws EasyPostException {
+        Map<String, Object> reportWithAdditionalColumnsParam = new HashMap<>();
+
+        List<String> additionalColumns = new ArrayList<>(Arrays.asList("from_name", "from_company"));
+
+        reportWithAdditionalColumnsParam.put("start_date", Fixture.reportStartDate());
+        reportWithAdditionalColumnsParam.put("end_date", Fixture.reportEndDate());
+        reportWithAdditionalColumnsParam.put("type", "shipment");
+        reportWithAdditionalColumnsParam.put("additional_columns", additionalColumns);
+
+        Report reportWithAdditionalColumns = Report.create(reportWithAdditionalColumnsParam);
+
+        assertTrue(reportWithAdditionalColumns instanceof Report);
+        assertTrue(reportWithAdditionalColumns.getId().startsWith("shprep_"));
```

Comment:
We don't need this ID test

Reply:
I believe from the discussion yesterday, we are going to have these two assertions, similar to other unit tests for `report`.

Reply:
Not exactly, we don't need to assert the ID since all we are testing is if the report gets created correctly. We are testing the ID elsewhere. We should assert that what comes back is a report (indicating success with the params passed) and then leaving a comment similar to what we do with batches saying we can't test this more in-depth because the report gets queued for generation and most likely isn't available quick enough for us to retrieve.

Reply:
sounds good, this will be addressed